Sync analytics error

Run node scripts/sync-analytics.mjs
node:internal/modules/cjs/loader:1210
  throw err;
  ^

Error: Cannot find module '/home/runner/work/gitly.sh/gitly.sh/scripts/sync-analytics.mjs'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Module._load (node:internal/modules/cjs/loader:1038:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.20.0
Error: Process completed with exit code 1.
0s

--- a/.github/workflows/sync-analytics.yml
+++ b/.github/workflows/sync-analytics.yml
@@ -24,12 +24,21 @@ jobs:
         with:
           node-version: '20'
       
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9
+      
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+      
       - name: Fetch and sync analytics
+        working-directory: scripts
         env:
           ANALYTICS_API_URL: ${{ secrets.ANALYTICS_API_URL }}
           ANALYTICS_API_KEY: ${{ secrets.ANALYTICS_API_KEY }}
         run: |
-          node scripts/sync-analytics.mjs
+          pnpm sync-analytics
       
       - name: Commit and push changes
         run: |

